packages/platforms/quanta/x86-64/x86-64-quanta-ix1b-rglbmc/onlp/builds/src/x86_64_quanta_ix1b_rglbmc/module/src/thermali.c

```diff
@@ -0,0 +1,33 @@
+/************************************************************
+ * <bsn.cl fy=2014 v=onl>
+ *
+ *           Copyright 2014 Big Switch Networks, Inc.
+ *
+ * Licensed under the Eclipse Public License, Version 1.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ *        http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the
+ * License.
+ *
+ * </bsn.cl>
+ ************************************************************
+ *
+ *
+ *
+ ***********************************************************/
+#include <onlp/platformi/thermali.h>
+#include "x86_64_quanta_ix1b_rglbmc_log.h"
+
+int
+onlp_thermali_init(void)
```

Comment:
@jonathantsai-qci Please remove all AIM_LOG_MSG statements. These are always output regardless of the logging level. 

Even though the BMC is managing the thermal plan the user/snmp etc all still need to be able to read the thermal information. Do you intend to provide support later?